pkg/helm/actions/install_chart_test.go

```diff
@@ -1,33 +1,46 @@
 package actions
 
 import (
+	"io/ioutil"
+	"testing"
+
 	"github.com/stretchr/testify/assert"
 	"helm.sh/helm/v3/pkg/action"
 	"helm.sh/helm/v3/pkg/chartutil"
 	kubefake "helm.sh/helm/v3/pkg/kube/fake"
 	"helm.sh/helm/v3/pkg/release"
 	"helm.sh/helm/v3/pkg/storage"
 	"helm.sh/helm/v3/pkg/storage/driver"
-	"io/ioutil"
-	"testing"
 )
 
 func TestInstallChart(t *testing.T) {
-	chart := "../testdata/influxdb-3.0.2.tgz"
-	store := storage.Init(driver.NewMemory())
-	actionConfig := &action.Configuration{
-		Releases:     store,
-		KubeClient:   &kubefake.PrintingKubeClient{Out: ioutil.Discard},
-		Capabilities: chartutil.DefaultCapabilities,
-		Log:          func(format string, v ...interface{}) {},
+	tests := []struct {
+		name  string
+		chart string
+	}{
+		{
+			name:  "valid chart",
```

Comment:
Also test invalid?  Ideally have 2-3 permutations with clear names to help us along later if something breaks.